utils/src/main/kotlin/Extensions.kt

```diff
@@ -340,3 +340,12 @@ fun TypeFactory.constructTreeSetType(elementClass: Class<*>): CollectionType =
  * Check whether the URI has a fragment that looks like a VCS revision.
  */
 fun URI.hasFragmentRevision() = fragment?.let { Regex("[a-fA-F0-9]{7,}$").matches(it) } == true
+
+/**
+ * Return a copy of this [URI] with the user info removed.
+ */
+@Suppress("UNUSED") // Intended to be used via scripting.
+fun URI.stripUserInfo() = run {
```

Comment:
Why an expression function with an "artificial" function body instead of a usual function?

Or, why not just `fun URI.stripUserInfo() = URI(scheme, null, host, port, path, query, fragment)`?

Reply:
And BTW, we already have `fun String.stripCredentialsFromUrl()` in this very file.

Reply:
> Or, why not just `fun URI.stripUserInfo() = URI(scheme, null, host, port, path, query, fragment)`?

I wanted to get rid of the ```magic number``` but changed it as you proposed.